qos-service-impl/src/test/java/com/palantir/atlasdb/qos/AtlasDbQosClientTest.java

```diff
@@ -18,54 +18,76 @@
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
 import static org.mockito.Mockito.when;
 
-import java.util.concurrent.TimeUnit;
-
-import org.jmock.lib.concurrent.DeterministicScheduler;
 import org.junit.Before;
 import org.junit.Test;
 
 import com.palantir.atlasdb.qos.client.AtlasDbQosClient;
 import com.palantir.atlasdb.qos.ratelimit.QosRateLimiter;
 
 public class AtlasDbQosClientTest {
+
+    private static final int ESTIMATED_BYTES = 10;
+    private static final int ACTUAL_BYTES = 51;
```

Comment:
Looks like there's a test waiting to be written about a second read going through even though `ACTUAL_BYTES > BYTE_LIMIT/2`, and then another one about a third read not being allowed.

Reply:
Yeah, that stuff is currently tested in `QosRateLimiterTest`, and the tests here just assert that we delegate to that class.

We do need ete tests for the whole setup here, which @hsaraogi is working on